1.0/docs/devguide/local-dom.md

```diff
@@ -8,7 +8,7 @@ subtitle: Developer guide
 
 {% include toc.html %}
 
-We call the DOM that an element is in charge of creating and managing its _local
+We call the DOM an element that is in charge of creating and managing its _local
```

Comment:
IMO, it reads better before the change. A `,` after managing might help.

The sentence would read better as a statement:
The DOM that an element creates and/or manages is called its _local DOM_.